prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/LA/LA-receiver-transform.yml

```diff
@@ -31,13 +31,103 @@ elements:
     value: [ '"LA-ELR"' ]
     hl7Spec: [ 'MSH-5-1' ]
 
+  - name: la-receiving-application-universal-id
+    condition: 'true'
+    value: [ '"LA-ELR"' ]
+    hl7Spec: [ 'MSH-5-2' ]
+
+  - name: la-receiving-application-universal-id-type
+    condition: 'true'
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-5-3' ]
+
   - name: la-receiving-facility-namespace-ID
     condition: 'true'
     value: [ '"LADOH"' ]
     hl7Spec: [ 'MSH-6-1' ]
 
+  - name: la-receiving-facility-universal-id
+    condition: 'true'
+    value: [ '"LADOH"' ]
+    hl7Spec: [ 'MSH-6-2' ]
+
+  - name: la-receiving-facility-universal-id-type
+    condition: 'true'
+    value: [ '"ISO"' ]
+    hl7Spec: [ 'MSH-6-3' ]
+
   - name: la-patient-race-coding-system
     resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
     condition: '%resource.code.exists()'
     value: [ '%resource.system.getCodingSystemMapping()' ]
-    hl7Spec: [ '%{PID}-10-3' ]
\ No newline at end of file
+    hl7Spec: [ '%{PID}-10-3' ]
+
+  - name: la-patient-county-codes
+    value: [ 'Bundle.entry.resource.ofType(Patient).address.district' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-11-6' ]
```

Comment:
This should be PID-11-9. Make sure to validate the correct hl7 spec location for the value even if copying from the repo as the initial value could be incorrect (in this case me, I put in the wrong thing for AK)

Reply:
Oh shoot, their email even specified that. Good catch!